crates/engine/tree/src/tree/root.rs

```diff
@@ -72,6 +85,78 @@ impl StdReceiverStream {
     }
 }
 
+/// Messages used internally by the state root task
+#[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
```suggestion
```

Reply:
clippy complains without this:
```
warning: variant `StateUpdate` is never constructed
  --> crates/engine/tree/src/tree/root.rs:91:5
   |
89 | pub(crate) enum StateRootMessage {
   |                 ---------------- variant in this enum
90 |     /// New state update from transaction execution
91 |     StateUpdate(EvmState),
   |     ^^^^^^^^^^^
```
will be fixed on #12639